### DISCUSSION TITLE
VPN connected but no internet

### DISCUSSION THREAD

**Discussion by raseltheone** [strongswan/strongswan#901] [Q&A]
server: /etc/ipsec.conf

# ipsec.conf - strongSwan IPsec configuration file
config setup
    charondebug="ike 1, knl 1, cfg 0"
    uniqueids=never

conn ikev2-vpn
    auto=add
    compress=no
    type=tunnel
    keyexchange=ikev2
    fragmentation=yes
    forceencaps=yes
    dpdaction=clear
    dpddelay=300s
    rekey=no
    left=%any
    leftid=142.44.205.91
    leftcert=server-cert.pem
    leftsendcert=always
    leftsubnet=0.0.0.0/0
    right=%any
    rightid=%any
    rightauth=eap-mschapv2
    rightsourceip=10.10.10.0/24
    rightdns=8.8.8.8,8.8.4.4
    rightsendcert=never
    eap_identity=%identity

**Comment by tobiasbrunner** [comment]
Please refer to #196.
